web/app/components/related-resources/add.ts

```diff
@@ -329,8 +329,11 @@ export default class RelatedResourcesAddComponent extends Component<RelatedResou
     const relatedHermesDocument = {
       googleFileID: attrs.objectID,
       title: attrs.title,
-      type: attrs.docType,
```

Comment:
We were incorrectly trying to save this attribute as `type` when the backend expects `documentType`